README.md

```diff
@@ -14,76 +14,54 @@
+2. Click on a filename with suffix `.ipynb` and the notebook will open in your browser.
+3.  Click on one of the two blue checkboxes in your browser window labelled either **Open in Colab** or **Open in Kaggle**: [colab](https://colab.research.google.com/notebooks/intro.ipynb#) and [kaggle](https://www.kaggle.com/) are websites providing free-of-charge platforms for running jupyter notebooks.
+4. Operate the notebook in your browser, editing, inserting or deleting cells as desired.
+5. Changes you make to the notebook will last no longer than your colab or kaggle session.
+ 
 
-pip
----
+## Install Python package
 
-    pip install tiatoolbox
+If you wish to use our programs, without developing them further, run the command<br/>
+`pip install tiatoolbox`<br/>
+To understand better how the programs work, study the jupyter notebooks referred to under the heading **Examples Taster**. For more information, you may also run<br/>
+`program_name --help`<br/>
 
-conda
------
-
-    conda env create -n tiatoolbox python=3.{6,7,8}
-    conda activate tiatoolbox
-    pip install tiatoolbox
-
-tiatoolbox --help
-=================
-
-    usage: tiatoolbox [-h] [--version] [--verbose VERBOSE]
-                     {read-bounds,
-                      save-tiles,
-                      slide_info,
-                      slide-thumbnail,
-                      stainnorm,
-                      }
-                     ...
-
-    positional arguments:
-      {slide_info}
-
-    read-bounds         usage: tiatoolbox read-bounds -h
-    save-tiles          usage: tiatoolbox save-tiles -h
-    slide-info          usage: tiatoolbox slide-info -h
-    slide-thumbnail     usage: tiatoolbox slide-thumbnail -h
-    stainnorm           usage: tiatoolbox stainnorm -h
-
-    optional arguments:
-      -h, --help            show this help message and exit
-      --version             show program`s version number and exit
-      --verbose             VERBOSE
+## Prepare for development
 
+Prepare a computer as a convenient platform for further development of the Python package `tiatoolbox` and related programs as follows.
+1. Open a `bash` terminal window<br/> 
+`cd <future-home-of-tiatoolbox>` <br/>
+2. Download a complete copy of the `tiatoolbox`.<br/>`conda clone https://github.com/TIA-Lab/tiatoolbox.git`<br/>
+3. `cd tiatoolbox`
+4. `conda env create -f requirements.conda.yml`<br/>
+creates the virtual environment `tiatoolbox`. Details in the text file `requirements.conda.yml`
+5. To use the packages installed in the environment, run the command:<br/>`conda activate tiatoolbox`
+6. `pip install tiatoolbox`<br/>installs the module and makes available the programs in the module.
```

Comment:
Please remove number 6.

Reply:
Deleted.
I included the word "perhaps" in the previous subsection.
If you wish to use our programs, **perhaps** without developing them further